include/boost/json/error.hpp

```diff
@@ -213,6 +213,28 @@ enum class error
 
     /// test failure
     test_failure,
+
+    //
+    // pointer errors
+    //
+
+    /// missing slash character before token reference
+    missing_slash,
+
+    /// invalid escape sequence
+    invalid_escape,
+
+    /// reference token is too large
+    token_too_large,
+
+    /// token should be a number but cannot be parsed as such
+    token_not_number,
+
+    /// current value is neither an object nor an array
+    value_is_scalar,
+
+    /// current value does not contain referenced value
+    not_found,
```

Comment:
trailing comma on the last element, is this valid C++?

Reply:
Yes https://eel.is/c++draft/dcl.enum#nt:enum-specifier

Reply:
Make sure Doxygen doesn't choke on it